examples/protein_language_modeling.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<p>That section is a bit dry! A couple of comments between each code cell would be nice!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>